LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/DownlinkMessageBuilder.cs

```diff
@@ -100,6 +102,7 @@ internal static class DownlinkMessageBuilder
                 }
 
                 tmst = rxpk.Tmst + CalculateTime(timeWatcher.GetReceiveWindow1Delay(loRaDevice), loRaDevice.ReportedRXDelay);
+                rxDelay = (ushort)timeWatcher.GetReceiveWindow1Delay(loRaDevice);
```

Comment:
Is this the LBS protocol Rx Delay? If yes could we give him a different name? We already have a rxdelay part of the LoRa Specification. I know it's not your doing but we already had issue with confusing those two and would avoid this happen in the future